recipes/google-cloud-aiplatform/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "google-cloud-aiplatform" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "89d2936985eaf71a52152fe62bae9d7554dec6948032526b99ec05a049d1ee39" %}
+{% set dev_url = "https://github.com/googleapis/python-aiplatform" %}
+{% set doc_url = "https://cloud.google.com/python/docs/reference/aiplatform/latest" %}
```

Comment:
If it's just used once or twice it's better to have this hardcoded. Makes it easier for the bot as well. Please add these three directly instead.

Reply:
Done.